codegen/tests/simple/simple_gencallback.go

```diff
@@ -41,3 +37,21 @@ func (c Callback) AddMiddleware(ctx context.Context, r chi.Router) {
 func (c Callback) BasePath() string {
 	return "/"
 }
+
+func (c CallbackWithMapError) MapError(ctx context.Context, cause error) *common.HTTPError {
+	return &common.HTTPError{HTTPCode: 500, Code: "1001", Description: "foo"}
```

Comment:
This should be passed in from the test rather than hardcoded. Add a field to the `Callback` and remove `CallbackWithMapError`:
```go
type Callback struct{
    MapError *common.HTTPError
}

func (c CallbackWithMapError) MapError(ctx context.Context, cause error) *common.HTTPError {
    return c.MapError
}
```

then just pass in your error to test:
```go
cb := Callback{ &common.HTTPError{ ... } }
``